Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.10-alpine as builder
+
+RUN apk --update upgrade \
+&& apk --no-cache --no-progress add git mercurial bash gcc musl-dev curl tar \
+&& rm -rf /var/cache/apk/*
+
+WORKDIR /go/src/github.com/letsencrypt/pebble
+COPY . .
+
+RUN go get -u github.com/jmhodges/clock \
+&& go get -u gopkg.in/square/go-jose.v2
+
+RUN go install ./...
```

Comment:
Could you make the Builder script also run the unit tests? Is that an anti-pattern?

Reply:
The purpose of a Dockerfile is not to replace a CI.
A Dockerfile must contain only what allows to build.